app/store.js

```diff
@@ -8,13 +8,26 @@ module.exports = ({
         useNewUrlParser: true
     }).then(() => {
         return {
+
+
             saveUser: (user) => {
                 return User.create(user)
                     .then((response) => {
-                        console.log("A new user has been created."); // now this is true since we are in the "then" part
-                        return response; // always do that in case the client calling this function is interested in the result from .create
+                        console.log("A new user has been created.");
+                        return response;
+                    })
+            },
+
+            findAllUsers: () => {
+                return User.find()
+                    .then((responce) => {
```

Comment:
`response` maybe?

Reply:
Yes, you're correct.  I noticed this when continuing to work after putting out this PR, the error got corrected in the new branch which I'm working on at the moment (find-single-user), which I hope to make a PR of today.

Thank you for checking the code out.